src/utilities.ts

```diff
@@ -394,85 +394,112 @@ function exeFilename(basename: string): string {
   else return basename;
 }
 
-function findVcastTools() {
-  // This function will check if vPython is on the path or if the
-  // VectorCAST Installation directory option is set
-  // In either of these "positive" cases it will setup the
-  // vPythonCommandToUse and clicastCommandtoUse global variables
-  // otherwise it will display and error.
+function findVcastTools():boolean {
+
+  // This function will set global paths to vpython, clicast and vcastqt
+  // by sequentially looking for vpython in the directory set via the 
+  // 1. extension option: "vcastInstallationLocation"
+  // 2. VECTORCAST_DIR
```

Comment:
When we allow someone to enter `vcastInstallationLocation` into the GUI, can we "cross reference" that with what's in `VECTORCAST_DIR`?

If you set `vcastInstallationLocation` to be something that is _different_ to `VECTORCAST_DIR`, you're going to get the message when `clicast` and friends complain about it being "different":

```
VectorCAST has detected that the launched executable differs from the location
specified by the environment variable VECTORCAST_DIR.
```

Reply:
Ya I already handle that in a few places, look for "annoying message"